src/main/java/kernel/controllers/VoltageController.java

```diff
@@ -0,0 +1,12 @@
+package kernel.controllers;
+
+/**
+ * Contains methods for controlling the TDK Lambda Power supply
+ */
+public interface VoltageController {
+    void setVoltage(Double newVoltage);
```

Comment:
Should this interface extend [FactoryBean](https://spring.io/blog/2011/08/09/what-s-a-factorybean) as well?